Nodes/config/routes.php

```diff
@@ -1,62 +1,34 @@
 <?php
 
-use Cake\Core\Configure;
 use Cake\Routing\RouteBuilder;
 use Cake\Routing\Router;
-use Croogo\Core\CroogoRouter;
 
-Router::plugin('Croogo/Nodes', function (RouteBuilder $routeBuilder) {
-    $routeBuilder->fallbacks();
-});
+Router::plugin('Croogo/Nodes', ['path' => '/'], function (RouteBuilder $routeBuilder) {
+    $routeBuilder->prefix('admin', function (RouteBuilder $routeBuilder) {
+        $routeBuilder->extensions(['json']);
+
+        $routeBuilder->connect('/nodes/:action/*', ['controller' => 'Nodes']);
+    });
+
+    $routeBuilder->connect('/', ['controller' => 'Nodes', 'action' => 'promoted']);
+    $routeBuilder->connect('/promoted/*', ['controller' => 'Nodes', 'action' => 'promoted']);
+    $routeBuilder->connect('/search/*', ['controller' => 'Nodes', 'action' => 'search']);
+
+    // Content types
+    $routeBuilder->connect('/blog', ['controller' => 'Nodes', 'action' => 'index', 'type' => 'blog']);
+    $routeBuilder->connect('/blog/archives/*', ['controller' => 'Nodes', 'action' => 'index', 'type' => 'blog']);
+    $routeBuilder->connect('/blog/:slug', ['controller' => 'Nodes', 'action' => 'view', 'type' => 'blog']);
+    $routeBuilder->connect('/blog/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term', 'type' => 'blog']);
 
-CroogoRouter::mapResources('Nodes.Nodes', [
-    'prefix' => '/:api/:prefix/',
-]);
-
-Router::connect('/:api/:prefix/nodes/lookup', [
-    'plugin' => 'Croogo/Nodes',
-    'controller' => 'Nodes',
-    'action' => 'lookup',
-], [
-    'routeClass' => 'Croogo\Core\Routing\Route\ApiRoute',
-]);
-
-// Basic
-CroogoRouter::connect('/', [
-    'plugin' => 'Croogo/Nodes', 'controller' => 'Nodes', 'action' => 'promoted'
-]);
-
-CroogoRouter::connect('/promoted/*', [
-    'plugin' => 'Croogo/Nodes', 'controller' => 'Nodes', 'action' => 'promoted'
-]);
-
-CroogoRouter::connect('/search/*', [
-    'plugin' => 'Croogo/Nodes', 'controller' => 'Nodes', 'action' => 'search'
-]);
-
-// Content types
-CroogoRouter::contentType('blog');
-CroogoRouter::contentType('node');
```

Comment:
Isn't it a idea to move it to the database? This because users are able to define custom contentTypes, I don't see any reason why this shouldn't work then like the blog and node.